lib/rules/key-spacing.js

```diff
@@ -197,13 +207,17 @@ module.exports = function(context) {
             }
 
             width = widths[i];
-
-            if (align === "value") {
+            if (isSingleLine(context.getSource(node))) { // Only one line
                 report(property, "key", whitespace.beforeColon, beforeColon);
-                report(property, "value", whitespace.afterColon, targetWidth - width);
-            } else { // align = "colon"
-                report(property, "key", whitespace.beforeColon, targetWidth - width);
                 report(property, "value", whitespace.afterColon, afterColon);
```

Comment:
This logic for checking only spacing is identical to [lines 252-3](https://github.com/IanVS/eslint/blob/52fa9e15ad83812256dc409045eddf6c36cb0448/lib/rules/key-spacing.js#L252-253) below.